Container Security/kernel-eks-implementation-guide.md

```diff
@@ -0,0 +1,242 @@
+
+ - Create new EKS cluster. It may take couple minutes before cluster is fully up and functioning.
+   ```
+   $ eksctl create cluster
+       --name demo-cluster --region eu-west-1 \
+       --managed
+   ```
+
+ - (optional) Verify that your local kubectl utility has been configured to connect to the cluster.
+   ```
+   $ kubectl cluster-info
+   Kubernetes control plane is running at https://EEAB38XXXXXXXXXXXXXXXXXXXXXXXXXX.sk1.eu-west-1.eks.amazonaws.com
+   CoreDNS is running at https://EEAB38XXXXXXXXXXXXXXXXXXXXXXXXXX.sk1.eu-west-1.eks.amazonaws.com/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+   To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+   ```
+   kubectl is command line tool that lets you control Kubernetes clusters. For configuration, kubectl looks for a file named config in the `$HOME/.kube` directory. This config was created previously by `eksctl create cluster` command and contains login information for your newly created cluster.
+
+### Step 3: Create ECR Container Repository
+
+ - Create container repository in AWS ECR. AWS ECR (Elastic Container Registry) is a cloud service providing a container registry. The bellow command creates a new repository in the registry and this repository will be subsequently used to store the Falcon Node sensor image.
+   ```
+   $ aws ecr create-repository --region eu-west-1 --repository-name falcon-node-sensor
+   {
+       "repository": {
+           "repositoryArn": "arn:aws:ecr:eu-west-1:123456789123:repository/falcon-node-sensor",
+           "registryId": "12345678912345",
+           "repositoryName": "falcon-sensor",
+           "repositoryUri": "123456789123.dkr.ecr.eu-west-1.amazonaws.com/falcon-node-sensor",
+           "createdAt": "2021-02-04T10:30:30+00:00",
+           "imageTagMutability": "MUTABLE",
+           "imageScanningConfiguration": {
+               "scanOnPush": false
+           },
+           "encryptionConfiguration": {
+               "encryptionType": "AES256"
+           }
+       }
+   }
+   ```
+
+ - Note the URI of the newly created repository to the environment variable for further use. Falcon Container Sensor will be available under this URI.
+   ```
+   $ FALCON_NODE_IMAGE_URI=123456789123.dkr.ecr.eu-west-1.amazonaws.com/falcon-node-sensor
+   ```
+
+### Step 3: Build falcon-node-sensor container image
+
+ - Provide OAuth2 Client ID and Client Secret for authentication with CrowdStrike Falcon platform. Establishing and retrieving OAuth2 API credentials can be performed at [falcon-console](https://falcon.crowdstrike.com/support/api-clients-and-keys). These credentials will only be used to download sensor, we recommend you create key pair that has permissions only for
+   ```
+   $ FALCON_CLIENT_ID=1234567890ABCDEFG1234567890ABCDEF
+   $ FALCON_CLIENT_SECRET=1234567890ABCDEFG1234567890ABCDEF
+   ```
+
+ - (optional) Provide name of Falcon Cloud you want to used
+   ```
+   $ FALCON_CLOUD=us-1
+   ```
```

Comment:
If this is optional, what is the default?

Reply:
Meaning new customers are /supposed/ to be in us-2. Should us-2 be the default (since that's where customers are heading), or should us-1 be default (since that's where most customers currently are)?